astropy/io/ascii/tests/data/cds/null/table1.dat

```diff
@@ -0,0 +1,2 @@
+Cr110   2108 6698.67 Al1 3.14  -1.65  58.0  -1.3  2.0  0.325
+Cr110   2108 6696.79 Al1 4.02  -1.42  29.5  -8.5  2.2  0.289
```

Comment:
```suggestion
Cr110   2108 6698.67 Al1  3.14  -1.65  58.0  -1.3  2.0  0.325
Cr110   2108 6696.79 Al1  4.02  -1.42  29.5  -8.5  2.2  0.289
```
Seems you have missed one column there, though for the tests it works as the actual numbers don't use the full format width.